pkg/kotsutil/embedded.go

```diff
@@ -40,9 +41,9 @@ func GetEmbeddedRegistryCreds(clientset kubernetes.Interface) (hostname string,
 	var err error
 
 	// kURL registry secret is always in the 'default' namespace
-	// Embedded cluster registry secret is always in the 'kotsadm' namespace
-
-	for _, ns := range []string{"default", "kotsadm"} {
+	// The Embedded Cluster registry secret is located in the 'kotsadm' namespace
+	// or in the application's namespace (named after the app slug in EC V3).
+	for _, ns := range []string{"default", "kotsadm", util.PodNamespace} {
```

Comment:
Would it make sense to have pod namespace as the first in this list? Then it would take precedence